src/components/App.css

```diff
@@ -37,3 +37,11 @@
   margin-top: 25px;
   margin-right: 20px;
 }
+
+/* Utils */
+.vertical-center {
+  top: 50%;
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);
```

Comment:
yeah we can remove this util for now... its not used
would be useful later on, i noticed we are using flexbox and other solutions for centering and that usually doesn't give a true center. originally I added this thinking I could center the `x` and then i realized it was a managed tree component ;P